packages/issuer/src/blockchain-facade/CertificationRequest.ts

```diff
@@ -79,14 +79,15 @@ export class CertificationRequest extends PreciseProofEntity implements ICertifi
             ? issuerWithSigner.requestCertificationFor(data, forAddress, isVolumePrivate)
             : issuerWithSigner.requestCertification(data, isVolumePrivate));
 
-        const { events } = await tx.wait();
+        const {
+            events: [certificationRequested]
+        } = await tx.wait();
 
-        request.id = Number(
-            events.find((log: BlockchainEvent) => log.event === 'CertificationRequested').topics[2]
-        );
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
not clear why we have this comment here

Reply:
Using `any` is by default treated as a error by linter, which is in 99.9% cases correct, this is an exemption and this comment make linter not throwing an error.